src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -73,6 +93,53 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
 
     return (
       <>
+        {(() => {
+          if (this.state.createdPipelineRun) {
+            return (
+              <InlineNotification
+                kind="success"
+                title="Successfully created a PipelineRun"
+                subtitle={
+                  <Link to={this.state.createdPipelineRunURL}>
+                    {this.state.createdPipelineRunName}
+                  </Link>
+                }
+                hideCloseButton={false}
+              />
+            );
+          }
+          return null;
+        })()}
+        <Button
+          iconDescription="Button icon"
+          renderIcon={Add}
+          onClick={() => this.toggleModal(true)}
+          style={{ float: 'right' }}
+        >
+          Create PipelineRun
+        </Button>
+        {(() => {
+          let namespace;
+          if (pipelineName) {
```

Comment:
move this above the return, it'll clean up the JSX so you can just have the `<CreatePipelineRun... />` directly without needing this function and all the associated brackets